packages/ag-charts-community/src/chart/series/seriesOptions.ts

```diff
@@ -27,7 +27,18 @@ export interface AgSeriesHighlightStyle {
     series?: AgSeriesHighlightSeriesStyle;
 }
 
-export interface AgBaseSeriesOptions<DatumType> {
+export interface AgBaseSeriesThemeableOptions {
+    /** Whether to include the series in the legend. */
+    showInLegend?: boolean;
+    /** The cursor to use for hovered area markers. This config is identical to the CSS `cursor` property. */
```

Comment:
Looks good but now some of these comments don't apply to all series types so need to be made generic

Reply:
Fixed